mandatory/3-playing-computer.js

```diff
@@ -7,12 +7,17 @@
   Answer the following questions:
 
   1. This program throws an error. Why? (If you can't find it, try executing it).
+    String b it's not declared and it's being called on line 32
   2. Remove the line that throws the error.
   3. What is printed to the console?
+    x = 2 and a = 6
```

Comment:
look for all the places where console.log() is called. Which values would be logged to the console if the script was run?